extensions/positron-r/src/tasks.ts

```diff
@@ -37,25 +38,29 @@ export async function getRPackageTasks(editorFilePath?: string): Promise<vscode.
 			task: 'r.task.packageCheck',
 			message: vscode.l10n.t('{taskName}', { taskName: 'Check R package' }),
 			rcode: 'devtools::check()',
-			package: 'devtools'
+			package: 'devtools',
+			envVars: null
 		},
 		{
 			task: 'r.task.packageInstall',
 			message: vscode.l10n.t('{taskName}', { taskName: 'Install R package' }),
 			rcode: 'pak::local_install(upgrade = FALSE)',
-			package: 'pak'
+			package: 'pak',
+			envVars: null
 		},
 		{
 			task: 'r.task.packageTest',
 			message: vscode.l10n.t('{taskName}', { taskName: 'Test R package' }),
 			rcode: 'devtools::test()',
-			package: 'devtools'
+			package: 'devtools',
+			envVars: await getEnvVars(['TESTTHAT_MAX_FAILS'])
```

Comment:
This needs some exception handling; if we fail to get the env vars for some reason, the result will be that no package tasks will be defined all. We either need to try/catch here or have `getEnvVars` avoid throwing in all circumstances (errors just logged and empty results returned safely)

Reply:
I think that is OK in this situation, because of how `getRPackageTasks()` is called when the command is executed, right? I think I would rather surface weird errors than swallow them, like for example this one I got in the midst of a weird debugging state:

![Screenshot 2024-09-09 at 7 56 01 PM](https://github.com/user-attachments/assets/9915d02a-e88d-469b-bc3c-bb4813d36b59)

To be clear, I'm not worried about this error in particular but would rather such things show up.

@jmcphers I know you approved the PR but I'll wait for one more response from you to make sure I am not misunderstanding the situation.

Reply:
OK, that looks good!